packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignUpRequest.kt

```diff
@@ -46,7 +46,11 @@ data class FlutterSignUpRequest(val map: HashMap<String, *>) {
                 }
                 authUserAttributes.add(AuthUserAttribute(AuthUserAttributeKey.custom(keyCopy), value))
             } else {
-                var t: Method = attributeMethods.asIterable().find { it.name.equals(convertSnakeToCamel(key)) } as Method;
+                // special case for amplify-android camelCase zoneinfo
+                if (key == "zoneinfo") {
+                    keyCopy = "zoneInfo"
+                }
+                var t: Method = attributeMethods.asIterable().find { it.name.equals(convertSnakeToCamel(keyCopy)) } as Method;
```

Comment:
This went from finding for any key in all of userAttributes, to finding `zoneInfo` or `key` which are the only two possible values in this else block. Is this accurate? Maybe it needs unit test